lib/relation-definition.js

```diff
@@ -3014,9 +3056,9 @@ EmbedsMany.prototype.remove = function(acInst, options, cb) {
   return cb.promise;
 };
 
-RelationDefinition.referencesMany = function referencesMany(modelFrom, modelTo, params) {
+RelationDefinition.referencesMany = function referencesMany(modelFrom, modelToRef, params) {
   params = params || {};
-  modelTo = lookupModelTo(modelFrom, modelTo, params, true);
+  var modelTo = lookupModelTo(modelFrom, modelToRef, params, true);
 
   var thisClassName = modelFrom.modelName;
   var relationName = params.as || i8n.camelize(modelTo.pluralModelName, true);
```

Comment:
same here: `params.reference || params.as || i8n.camelize(modelTo.pluralModelName, true);` if we decide to use reference